contracts/oracles/composite/CompositeMultiOracle.sol

```diff
@@ -31,74 +35,119 @@ contract CompositeMultiOracle is IOracle, AccessControl {
         }
     }
 
-    /// @dev Set or reset an price path
-    function setPath(bytes6 base, bytes6 quote, bytes6[] memory path)
-        external auth
-    {
-        bytes6[] memory reverse = new bytes6[](path.length);
+    /// @notice Set or reset an price path and its reverse path
+    /// @param base Id of base token
+    /// @param quote Id of quote token
+    /// @param path Path from base to quote
+    function setPath(
+        bytes6 base,
+        bytes6 quote,
+        bytes6[] memory path
```

Comment:
Should this be `calldata`? Or not because we are assigning it to `paths[base][quote]` below?

Reply:
Good catch.  I didnt see where we assigning it below, and no tests broke when i switched it so 👍